### ISSUE TITLE
issue with viewing settings on firefox browser 

### ISSUE THREAD

**Issue by gareth-d85** [freeCodeCamp/freeCodeCamp#16632]
#### Issue Description
When I click on the setting button the page looks blank but if you scroll to the right the setting are there.
I tested in my chrome browser and the issue id not happen

#### Browser Information

* Browser Name, Version: Firefox dev edition, 59.0b6 (64-bit)
* Operating System: Windows 10
* Mobile, Desktop, or Tablet: 

```
#### Screenshot

https://screenshots.firefox.com/hJagih97ne1Z5jCc/beta.freecodecamp.org

**Comment by gareth-d85** [comment]
setting page bug

**Comment by jkao1** [comment]
Yeah...I get this on Firefox Dev Edition and Firefox. I've tracked down the bug and will open a PR soon.

**Comment by gareth-d85** [comment]
Is it a simple fix?

**Comment by jkao1** [comment]
I wish I could say yes, but since the CSS in freeCodeCamp is not scoped, any simple changes require a great amount of testing.

**Comment by cairotx** [comment]
I would uninstall firefox and reinstall clean version

**Comment by gareth-d85** [comment]
I have tried uninstalling firefox but still no change.

**Comment by Bouncey** [comment]
Closed via [comment](https://github.com/freeCodeCamp/freeCodeCamp/pull/16633#issuecomment-363369568)
